demo/package.json

```diff
@@ -33,6 +34,9 @@
         "webpack-cli": "^5.1.4"
     },
     "dependencies": {
+        "@emotion/react": "^11.11.3",
```

Comment:
It doesn't seem like you're actually using `@emotion`, in which case it's good to remove from this PR.